cli.ts

```diff
@@ -430,59 +426,61 @@ const initProject = (
 		map(_ => i18n.__('bootstrapMsg')),
 	);
 
+const runScaffoldPostInit = (scaffoldDir: SanitizedAbsPath.t): Future<TaqError.t, SanitizedAbsPath.t> => {
+	const scaffoldConfigAbspath = SanitizedAbsPath.create(`${scaffoldDir}/scaffold.json`);
+	return pipe(
+		readJsonFile<ScaffoldConfig.t>(scaffoldConfigAbspath),
+		map(logInput('Scaffold Config')),
+		coalesce(_ => ({}) as ScaffoldConfig.t)(identity),
+		chain(scaffoldConfig =>
+			typeof scaffoldConfig === 'object' && scaffoldConfig.postInit
+				? pipe(
+					exec(`${scaffoldConfig.postInit!} 2>&1`, {}, true, scaffoldDir),
```

Comment:
What is the `!` at the end for?

Reply:
Null coercion

Reply:
What happens if `postInit` doesn't exist? Are we confident it does?

Reply:
Hmm nvm it looks fine to me here